packages/editor/src/client/api/comment-rating.graphql

```diff
@@ -0,0 +1,46 @@
+query RatingSystem {
```

Comment:
To extend the RatingSystem easier in the future, we should eventually pass already a param. For now it would be "default".
It would help to avoid breaking changes for all the websites out there.

Reply:
I thought too but then decided against it, as the breaking change would happen as you would need to get the id from the article/page. Making the breaking change break at compile time is IMO better than at runtime.

What do you think?